docs/design/lexical_conventions/words.md

```diff
@@ -29,6 +29,10 @@ the precise rules are decided; see the
 [Unicode source files](/proposals/p0142.md#characters-in-identifiers-and-whitespace)
 proposal.
 
+The Carbon source files (including comments and string literals) are required to
```

Comment:
```suggestion
Carbon source files, including comments and string literals, are required to
```

I think this reads better without the "the", and we generally prefer commas for parentheticals.

Reply:
ohk. I have updated it